tests/ltp/tests

```diff
@@ -0,0 +1,1222 @@
+#/ltp/testcases/kernel/syscalls/abort/abort01
```

Comment:
@mikbras , @salsal97 enabled more tests in ltp_enabled.txt in PR #165 which is merged. Please make sure these new tests are enabled in tests/ltp/tests

Reply:
I backed out my tests/ltp change. Thank you.